test/lib/server-http1.ts

```diff
@@ -310,11 +314,11 @@ export class ServerHttp1 extends TypedServer< HttpServer | HttpsServer >
 }
 
 export async function makeServer( opts: ServerOptions = { } )
-: Promise< { server: Server; port: number | null; } >
+: Promise< { server: Server; port: number | null; receivedCookies: Array<string> } >
```

Comment:
[minor]: As i described before use constructors as types is not a good idea.

```suggestion
: Promise< { server: Server; port: number | null; receivedCookies: string[] } >
```